mono/mini/mini-posix.c

```diff
@@ -1049,9 +1058,8 @@ dump_native_stacktrace (const char *signal, MonoContext *mctx)
 					} else {
 						// Remove
 						g_async_safe_printf("\nThe MERP upload step has succeeded.\n");
-						mono_summarize_timeline_phase_log (MonoSummaryDone);
 					}
-
+					mono_summarize_timeline_phase_log (MonoSummaryDone);
```

Comment:
Why is this here? if the MERP upload fails, we don't want to make the state as done.

Reply:
It was from when I thought DumpStateTotal used this function and so this had to end with re-set state. Reverted